tests/core/models/test_edition.py

```diff
@@ -325,6 +328,67 @@ def test_calculate_presentation_author(self, db: DatabaseTransactionFixture):
         assert "Kelly Accumulator, Bob A. Bitshifter" == wr.author
         assert "Accumulator, Kelly ; Bitshifter, Bob" == wr.sort_author
 
+    def test_calculate_presentation_very_long_author(
+        self, db: DatabaseTransactionFixture
+    ):
+        authors = []
+
+        # author names should be unique and not similar to ensure that the
+        # test mirrors the types of long author lists we'd expect in real data.
+        def generate_random_random_author():
+            return "".join(
+                random.choices(
+                    string.ascii_uppercase + string.ascii_lowercase + string.digits,
+                    k=25,
+                )
+            )
+
+        for i in range(0, 500):
+            author, ignore = db.contributor(sort_name=generate_random_random_author())
```

Comment:
Authors usually have at least a given name and a family name. If we're trying to hew closely to what we see in the real work, should this be:

```
author, ignore = db.contributor(
    ", ".join(
        sort_name=generate_random_random_author(),
        sort_name=generate_random_random_author(),
    )
)

Reply:
Fixed.